docs/reference/search/explain.asciidoc

```diff
@@ -135,8 +127,7 @@ This will yield the same result as the previous request.
     Controls on which shard the explain is executed.
 
```

Comment:
"Controls which", or maybe just "The shard where you want to run the explain operation."